repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/Authorisation.java

```diff
@@ -121,4 +124,25 @@ public String getCrudAsString() {
     public void setCrudAsString(final String crudAsString) {
         this.crudAsString = crudAsString;
     }
+
+    @Transient
+    public boolean hasLessAccessThan(Authorisation other) {
```

Comment:
Can we name it `hasLowerAccessThan` to be consistent with the data store codebase?